api/src/utils/apiHelper.ts

```diff
@@ -5,11 +5,63 @@ import { inspect } from "util";
 import { IDataResponse } from "../models/api/IDataResponse";
 import { IResponse } from "../models/api/IResponse";
 import { ISignedResponse } from "../models/api/ISignedResponse";
+import { INftDetailsResponse } from "../models/api/stardust/INftDetailsResponse";
 import { IHttpRequest } from "../models/app/IHttpRequest";
 import { IHttpResponse } from "../models/app/IHttpResponse";
 import { IRoute } from "../models/app/IRoute";
 import { IConfiguration } from "../models/configuration/IConfiguration";
 
+/**
+ * Get mock nft detils
+ * @returns nft details.
+ */
+export function getNftDetails(): INftDetailsResponse {
```

Comment:
Can we call this `getMockNftDetails` and move it inside file `api/src/routes/stardust/nftDetails/get.ts`, so we have everything there, and there are no changes to this file.

Reply:
Done.